File: Helios.Cont.Web/Models/SunatContribuyente.cs
using Newtonsoft.Json;
using Newtonsoft.Json.Linq;


namespace Helios.Cont.Web.Models
{
    public partial class SunatContribuyente
    {
        [JsonProperty("ruc")]
        public long Ruc { get; set; }

        [JsonProperty("nombre_o_razon_social")]
        public string NombreORazonSocial { get; set; }

        [JsonProperty("estado_del_contribuyente")]
        public string EstadoDelContribuyente { get; set; }

        [JsonProperty("condicion_de_domicilio")]
        public string CondicionDeDomicilio { get; set; }

        [JsonProperty("ubigeo")]
        public int Ubigeo { get; set; }

        [JsonProperty("tipo_de_via")]
        public string TipoDeVia { get; set; }

        [JsonProperty("nombre_de_via")]
        public string NombreDeVia { get; set; }

        [JsonProperty("codigo_de_zona")]
        public string CodigoDeZona { get; set; }

        [JsonProperty("tipo_de_zona")]
        public string TipoDeZona { get; set; }

        [JsonProperty("numero")]
        public int Numero { get; set; }

        [JsonProperty("interior")]
        public string Interior { get; set; }

        [JsonProperty("lote")]
        public string Lote { get; set; }

        [JsonProperty("dpto")]
        public string Dpto { get; set; }

        [JsonProperty("manzana")]
        public string Manzana { get; set; }

        [JsonProperty("kilometro")]
        public string Kilometro { get; set; }

        [JsonProperty("direccion")]
        public string Direccion { get; set; }
    }
}

File: Helios.Cont.UI.WinForm/PMO/Clases/SunatContribuyente.vb
Imports System
Imports System.Collections.Generic
Imports Newtonsoft.Json
Imports Newtonsoft.Json.Linq
Public Class SunatContribuyente
    <JsonProperty("ruc")>
    Public Property Ruc As Long

    <JsonProperty("nombre_o_razon_social")>
    Public Property NombreORazonSocial As String

    <JsonProperty("estado_del_contribuyente")>
    Public Property EstadoDelContribuyente As String

    <JsonProperty("condicion_de_domicilio")>
    Public Property CondicionDeDomicilio As String

    <JsonProperty("ubigeo")>
    Public Property Ubigeo As Integer

    <JsonProperty("tipo_de_via")>
    Public Property TipoDeVia As String

    <JsonProperty("nombre_de_via")>
    Public Property NombreDeVia As String

    <JsonProperty("codigo_de_zona")>
    Public Property CodigoDeZona As String

    <JsonProperty("tipo_de_zona")>
    Public Property TipoDeZona As String

    <JsonProperty("numero")>
    Public Property Numero As Integer

    <JsonProperty("interior")>
    Public Property Interior As String

    <JsonProperty("lote")>
    Public Property Lote As String

    <JsonProperty("dpto")>
    Public Property Dpto As String

    <JsonProperty("manzana")>
    Public Property Manzana As String

    <JsonProperty("kilometro")>
    Public Property Kilometro As String

    <JsonProperty("direccion")>
    Public Property Direccion As String
End Class
